loguru_mypy/__init__.py

```diff
@@ -23,6 +24,8 @@
 )
 import typing_extensions as te
 
+RECORD_ARG_REGEX = re.compile('record\\[["|\'](?P<arg>.*)["|\']\\]')  # type: te.Final
```

Comment:
```suggestion
RECORD_ARG_REGEX = re.compile('record\\[["|\'](?P<arg>[^\\]]*)["|\']\\]')  # type: te.Final
```
That would solve the nesting issue I guess.

Reply:
Who knows...only the oracle...LOL :dagger: 
I will check it out, but it looks as if it should work.

Reply:
This one works, I will push this change along with everything else.
Cheers to you for that.